apps/design/backend/src/app.ts

```diff
@@ -674,6 +676,25 @@ export function buildApp(context: AppContext): Application {
     res.send(grout.serialize(userInfo));
   });
 
+  app.get('/files/:orgId/:fileName', async (req, res) => {
+    const user = assertDefined(context.auth.userFromRequest(req));
+    const userOrg = await context.auth.org(user.org_id);
+    if (!userOrg) {
+      res.status(500).send('No org found for user');
+      return;
+    }
+
+    const { orgId, fileName } = req.params;
+    if (orgId !== userOrg.id && userOrg.id !== votingWorksOrgId()) {
+      res.status(404).send('File not found');
+    }
```

Comment:
I'm siloing files by org, but still making it such that Vx users can access any org's files.